docs/docs/README.md

```diff
@@ -4,24 +4,72 @@ id: intro
 slug: /intro
 sidebar_position: 1
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { Gear, Plus } from "@phosphor-icons/react";
 
 
 ![Banner](/img/banner.png)
 
 ---
 
 > Welcome to Butterfly, the opensource note app.
-> Use the sidebar to navigate through the wiki.
+
+Butterfly is a note taking up that is aiming for unified experience across devices. Butterfly's approach is for minimalistic view of the drawing canvas while maximizing the user's control over the app.
 
 ## Getting started
+> You can use the sidebar to navigate quicly through the wiki.
```

Comment:
```suggestion
> You can use the sidebar to navigate quickly through the wiki.
```

Reply:
Oh I didn't saw it while reviewing.
And please keep a line above and below free for better reading